controllers/databaseuser_controller.go

```diff
@@ -94,8 +98,10 @@ func (r *DatabaseUserReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 		if len(errs) == 0 {
 			if updated {
+				r.Recorder.Event(&user, corev1.EventTypeNormal, "Updated", "database user update succeeded")
```

Comment:
I wonder if the reason should be DBUser specific or should follow k8s conventions as much as possible.